### PR TITLE
Add ExUnitProperties.pick/1

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: lib/ex_unit_properties.ex [hunk 1]
```diff
@@ -471,4 +471,28 @@ defmodule ExUnitProperties do
...
+
+  ## Examples
+
+      ExUnitProperties.pick(StreamData.integer())
+      #=> -21
+
+  """
+  @spec pick(StreamData.t(a)) :: a when a: term()
+  def pick(data) do
+    seed = ExUnit.configuration()[:seed]
```
Comment (josevalim) [inline lib/ex_unit_properties.ex]: Let's use the new seed in ExUnit v1.6?

Comment (whatyouhide) [reply lib/ex_unit_properties.ex]: You mean the seed in the pdict? I guess that would make more sense?

**Comment by josevalim** [comment]
Yeah. The question is what to do when they call it outside of the test
process. Raise, right?
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by josevalim** [comment]
We cannot use the global seed because then all tests will likely pick the
same value :P
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

**Comment by whatyouhide** [comment]
@josevalim updated :)

**Pull request merged**